README.md

```diff
@@ -10,24 +10,40 @@ This is the code behind: https://www.digitale-gesellschaft.ch/auskunftsbegehren
 Please open issues here for anything you find not working correctly with the Application. If you miss any organisation or company in the data, feel free to open Pull Requests at the [Data Repository](https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren-Data).
 
 ## Develop
-### Setup local development environment
-*Note that you will need to have [Node.js](https://nodejs.org) installed.*
+
+### Prerequisites
+
+* Install [Node.js LTS](https://nodejs.org)
+* Create self signed SSL Certificate for localhost, required for `npm run dev`. Recommended CLI: [mkcert](https://github.com/FiloSottile/mkcert)
+
+```bash
+mkcert -install && \
+mkcert localhost 127.0.0.1 && \ 
+mv localhost+1-key.pem local.key && \ 
+mv localhost+1.pem local.cert
+```
```

Comment:
This is not necessary and was actually removed with https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren/commit/8cd97cceff17cb8284df96fb4e039a08a24cdc89 because `localhost` is considered a secure origin if it resolves to a loopback device by all browsers.

Reply:
When I `npm run dev` rollup requires these certs (for whatever reason they are still referenced in rollup config). This is why this part of the readme should be kept, or the rollup config must change.

Reply:
Fixed in https://github.com/DigitaleGesellschaft/Datenauskunftsbegehren/commit/aaeb683f79aa407c14310ebcce5f2b99fe2f5283